simulation/traffic_simulator/include/traffic_simulator/entity/ego_entity.hpp

```diff
@@ -92,6 +95,10 @@ class EgoEntity : public VehicleEntity
 
   void requestAssignRoute(const std::vector<geometry_msgs::msg::Pose> &) override;
 
+  auto requestFollowTrajectory(
+    const std::shared_ptr<traffic_simulator_msgs::msg::PolylineTrajectory> & parameter)
+    -> void override;
```

Comment:
The definition of this method in the API has been removed, the method is forwarded to the entity_manager and then to the specific entity. After the changes,  this method is intentionally called for Ego - the ZeroMQ client call is implemented in it. I moved this method to `ego_entity` because now the Ego methods: `requestSpeedChange` and setVelocityLimit also use the `ZeroMQ` client.

![image](https://github.com/tier4/scenario_simulator_v2/assets/121798334/38f233dd-d5c4-4793-a36f-9ce77b9e9ec3)

Reply:
_NOTE: See https://github.com/tier4/scenario_simulator_v2/pull/1163#discussion_r1452927183._